### ISSUE TITLE
OpenVPN Advanced Options checkboxes not showing in some cases

### ISSUE THREAD

**Issue by mikebell** [snwh/paper-gtk-theme#63]
![](http://i.imgur.com/6BPpRbk.png)

All the options on this dialogue have tick boxes but only 2 of them are showing up.

**Comment by snwh** [comment]
Can you confirm whether or not this has been fixed?

**Comment by mikebell** [comment]
Yes this still occurs. Although the behaviour has changed slightly, if I tick then untick the boundry box stays until the window loses focus.
